models/airdrop/airdrop_schema.yml

```diff
@@ -30,9 +30,13 @@ models:
       - &project
         name: project
         description: "Project name"
-      - &airdrop_identifier
-        name: airdrop_identifier
-        description: "Airdrop identifier name"
+        tests:
+          - relationships:
+              to: ref('airdrop_info')
+              field: project
```

Comment:
the screenshot you sent is broken, or at least for me. from what i can see in the gh action, this new test is failing. as for why, you'll have to run `dbt compile`, then grab test query output and run on dune to see results returned and debug from there. you can find the test query in a path similar to: `target/compiled/models/<path to spell>/schema/`

Reply:
exact name of test also in gh action logs